sp800-63b/sec5_authenticators.md

```diff
@@ -201,7 +201,28 @@ The verifier SHALL use approved encryption and SHALL utilize an authenticated pr
 
 If the authenticator output or activation secret has less than 64 bits of entropy, the verifier SHALL implement a throttling mechanism that effectively limits the number of failed authentication attempts an attacker can make on the subscriber’s account as described in [Section 5.2.2](#throttle). A biometric activation factor SHALL meet the requirements of [Section 5.2.3](#biometric_use), including limits on number of successive authentication failures.
     
-#### 5.1.6. Single Factor Cryptographic Devices
+#### 5.1.6. Single Factor Cryptographic Software
+
+<div class="text-left" markdown="1">
+<table style="width:100%">
+  <tr>
+    <td><img src="sp800-63b/media/Single-factor-software-crypto.png" alt="authenticator" style="width: 100px;height: 100px"/></td>
+    <td>A single factor software cryptographic authenticator is a cryptographic key stored on disk or some other “soft” media. Authentication is accomplished by proving possession and control of the key. The authenticator output is highly dependent on the specific cryptographic protocol, but it is generally some type of signed message. The SF software cryptographic authenticator is <i>something you have</i>.</td> 
+  </tr>
+  </table>
+  </div>
+
+
+
+##### 5.1.6.1. Single Factor Cryptographic Software Authenticators
+
+Single factor software cryptographic authenticators encapsulate a secret key that is unique to the authenticator. The key SHALL be stored in the most secure storage available on the device (e.g., keychain storage, trusted platform module, or trusted execution environment if available). The key SHALL be strongly protected against unauthorized disclosure by the use of access controls that limit access to the key to only those software components on the device requiring access.
```

Comment:
my own ignorance, 'secret key' is because this can be symmetric or asymmetric?  So u addressed earlier comment about TEE, but TPM is hardware.  I don't think we can say this.  Even the crypto ops on a smart card are done with a software applet on the chip...